### ISSUE TITLE
Validation rule checking for update.ps1

### ISSUE THREAD

**Issue by pascalberger** [chocolatey/package-validator#159]
A validation rule which checks for the existence of an AU update script (`update.ps1`) would be nice.

**Comment by gep13** [comment]
I tend to agree with this, however, I don't think it would be a rule that would fail the validation process, unless we inspect the file in question for something specifically that points to AU being used.  i.e. having a file called update.ps1 in the package itself wouldn't be enough, would it?

**Comment by pascalberger** [comment]
Better than a filecheck would be to check for a file containng an `import-module au` statement. But checking for `update.ps1` and raise a suggestion might also work.

**Comment by gep13** [comment]
Ah, good suggestion.  I think the presence of the `update.ps1` and usage of `import-module au` would be enough to fail validation, and to push it back to the maintainer.
